spec/shoryuken/fetcher_spec.rb

```diff
@@ -28,12 +28,12 @@
 
       Shoryuken.sqs_client_receive_message_opts[group] = { wait_time_seconds: 10 }
 
-      expect(queue).to receive(:receive_messages).with(
+      expect(queue).to receive(:receive_messages).with({
```

Comment:
why does this is a problem in ruby 3? 🤔

Reply:
I'm probably having this same problem, and the test fails in my environment (ruby 3.0.0)
https://github.com/lsegal/yard/pull/1433

Reply:
@andreleoni Keyword arguments are fully separated from hashes in ruby 3.x. https://rubyreferences.github.io/rubychanges/3.0.html#keyword-arguments-are-now-fully-separated-from-positional-arguments